core/storage/btree.rs

```diff
@@ -5395,18 +5425,24 @@ impl PageStack {
 
 /// Used for redistributing cells during a balance operation.
 struct CellArray {
-    cells: Vec<&'static mut [u8]>, // TODO(pere): make this with references
-
-    number_of_cells_per_page: [u16; 5], // number of cells in each page
+    /// The actual cell data.
+    /// TODO(pere): make this with references
+    cell_data: Vec<&'static mut [u8]>,
```

Comment:
Nit: we use payload a lot more through the codebase so I prefer that to data.
```suggestion
    cells_payload: Vec<&'static mut [u8]>,
```